### DISCUSSION TITLE
X修改服務條款，上傳圖片影片將無條件成AI素材

### DISCUSSION THREAD

**Discussion by christaaah** [1101-GS4719-C/Python-Community-GS4719-C#2220] [General]
我雖然平時不用X，但是還是偶爾會上X看一下Vtuber的推文，最近看到了以下報導
[https://www.4gamers.com.tw/news/detail/67883/x-takes-uploaded-meida-as-ai-training-element](url)
得知了X即將讓用戶上傳之圖片、影片成為他們訓練AI的素材，這樣在X上發圖的畫師或影片創作者用心良苦製作的作品就會被拿來學習，畫師們各個具有自己獨特風格的畫風會被模仿，讓創作者們紛紛其他平台，例如Bluesky，甚至因為這種風潮造成Bluesky當掉沒辦法登入。
我認為雖然創作者們是利用了X這個平台可以發布他們的作品，但是不代表可以讓平台隨意使用它們，目前服務條款還沒生效，所以也不清楚實行後的情況，希望X可以給予創作者們回應，不要讓他們努力的結果遭到損害。

**Comment by 112121py** [comment]
我覺得這項條款如果是強迫執行那會非常不尊重所有創作者，如果官方真的想要執行，應該要在用戶完全知情的情況下，進入X的時候，詢問用戶是否同意給官方當成訓練素材，不然的話，X的使用量應該會大量下降非常多，以後就看不到各種好看的圖了qwq

**Comment by Chattenoire** [comment]
基本上這就是網路平台私有化的結果；擁有者，現在是Elon Musk，就可以獨裁地拿這個平台做任何他想做的事
原先的twitter常常有荒謬的爭吵沒錯，但至少是民主的、甚至是民粹的，現在，就是獨裁。
